cypress.json

```diff
@@ -3,5 +3,7 @@
   "viewportHeight": 568,
   "viewportWidth": 320,
   "projectId": "8r2xra",
-  "defaultCommandTimeout": 10000
+  "defaultCommandTimeout": 10000,
+  "supportFile": "cypress/support/commands.js",
+  "pluginsFile": false
```

Comment:
I deleted `support/index.js`, all it has was importing this `commands.js`

Also I deleted `plugins` folder (when dirdoc suggests me to write documentation for it lmao)

If you don't specify pluginsFile: false, I believe cypress will regenerate them.